app/main/plugins/plugins.spec.js

```diff
@@ -1,9 +1,10 @@
+jest.dontMock('electron');
```

Comment:
This is probably not needed since `electron` is not part of the `__mocks__` folder just yet
anyway, I'd just leave it for now